README.md

```diff
@@ -16,20 +16,20 @@ Zeebe provides visibility into and control over business processes that span mul
 * Export process data for monitoring and analysis
 * Engage with an active community
 
-[Learn more at zeebe.io](https://zeebe.io)
+[Learn more at camunda.com](https://camunda.com/products/cloud/workflow-engine/)
```

Comment:
It would be great if that page can also direct developers back to our github (specifically for contributing). A user reported he ended up clicking the `How to contribute` link in the footer on that page, which leads to CamundaBPM related contributions.

Reply:
This is how this started. We are asking marketing to change the link from the landing page to the GitHub repo. So we want the README to make sense if you arrive from that route.